### PR TITLE
refactor: optimize project input name and navigation spacing

### PR DESCRIPTION
**Pull request by J1za**
## Linear Ticket
https://linear.app/autokitteh/issue/UI-722/top-navigation-bar-design-fine-tuning
## What type of PR is this? (check all applicable)

-   [ ] 💡 (feat) - A new feature (non-breaking change which adds functionality)
-   [ ] 🔄 (refactor) - Code Refactoring - A code change that neither fixes a bug nor adds a feature
-   [ ] 🐞 (fix) - Bug Fix (non-breaking change which fixes an issue)
-   [ ] 🏎 (perf) - Optimization
-   [ ] 📄 (docs) - Documentation - Documentation only changes
-   [ ] 📄 (test) - Tests - Adding missing tests or correcting existing tests
-   [ ] 🎨 (style) - Styles - Changes that do not affect the meaning of the code (white-space, formatting, missing semi-colons, etc)
-   [ ] ⚙️ (ci) - Continuous Integrations - Changes to our CI configuration files and scripts (example scopes: Travis, Circle, BrowserStack, SauceLabs)
-   [ ] ☑️ (chore) - Chores - Other changes that don't modify src or test files
-   [ ] ↩️ (revert) - Reverts - Reverts a previous commit(s).

### CODE REVIEW

File: src/components/organisms/topbar/project/projectName.tsx [hunk 1]
```diff
@@ -77,50 +77,42 @@ export const ProjectTopbarName = () => {
...
+				title={t("topbar.rename")}
+				value={project?.name}
+			/>
+
+			<ErrorMessage className="-bottom-3.5 text-xs">
+				{!isNameValid ? t("nameRequired", { ns: "errors" }) : null}
+			</ErrorMessage>
+
+			<span className="flex items-center font-fira-code font-semibold text-gray-500">
+				ID
```
Comment (RonenMars) [inline src/components/organisms/topbar/project/projectName.tsx:113]: i18n

File: tailwind.config.cjs [hunk 2]
```diff
@@ -22,6 +22,9 @@ module.exports = {
       spacing: {
         9.5: "2.375rem",
         '3px': '3px',
+        50: "50px",
+        245: "245px",
+        350: "500px",
```
Comment (RonenMars) [inline tailwind.config.cjs:27]:
```suggestion
        350: "350px",
```

File: src/components/organisms/topbar/project/projectName.tsx (hunk 1 above)
Comment (J1za) [reply src/components/organisms/topbar/project/projectName.tsx]: added

File: tailwind.config.cjs (hunk 2 above)
Comment (J1za) [reply tailwind.config.cjs]: changed

File: src/components/organisms/topbar/project/projectName.tsx [hunk 3]
```diff
@@ -12,15 +12,18 @@ import { useProjectStore, useToastStore } from "@store";
...
 	const { projectId } = useParams();
 	const { getProject, renameProject } = useProjectStore();
 	const [isNameValid, setIsNameValid] = useState(true);
 	const [project, setProject] = useState<Project>();
+	const [isEditing, setIsEditing] = useState(false);
 	const { t } = useTranslation(["projects", "buttons"]);
 	const { t: tErrors } = useTranslation("errors");
 	const inputClass = cn(
-		"min-w-3 rounded bg-transparent p-0 text-xl font-bold leading-6 leading-tight outline outline-0",
+		"min-w-3 rounded bg-transparent p-0 text-xl font-bold leading-6 leading-tight outline outline-0 max-w-120 transition focus:bg-gray-1200",
```
Comment (RonenMars) [inline src/components/organisms/topbar/project/projectName.tsx:26]: Please make it `max-width` 240 on normal screens
and `max-width` 160 on screen smaller than 1600px

**Review by RonenMars — COMMENTED** [review]
Please place the edit button outside the box, maybe on the left
image

File: src/components/organisms/topbar/project/projectName.tsx (hunk 3 above)
Comment (J1za) [reply src/components/organisms/topbar/project/projectName.tsx]: changed

**Review by RonenMars — APPROVED** [review]

File: src/components/organisms/topbar/project/projectName.tsx [hunk 4]
```diff
@@ -77,50 +78,64 @@ export const ProjectTopbarName = () => {
...
 		setIsNameValid(validateName(newName));
 	};
 
 	return (
-		<div className="flex items-center py-2">
-			<div className="relative flex items-center gap-3 font-fira-code text-gray-500">
-				<span
+		<div className="flex items-center gap-3 py-2 font-fira-code">
+			{isEditing ? (
+				<input
```
Comment (RonenMars) [inline src/components/organisms/topbar/project/projectName.tsx:99]: Use `Input` component

Comment (J1za) [reply src/components/organisms/topbar/project/projectName.tsx:99]: changed

**Pull request merged**